core/src/main/java/org/apache/accumulo/core/spi/scan/ConfigurableScanServerSelector.java

```diff
@@ -249,7 +249,7 @@ protected static class Profile {
     int busyTimeoutMultiplier;
     String maxBusyTimeout;
     String group = ScanServerSelector.DEFAULT_SCAN_SERVER_GROUP_NAME;
-    String timeToWaitForScanServers = "0s";
+    String timeToWaitForScanServers = "100d";
```

Comment:
I wonder if it would be better to use a negative number for the wait time when disabling. This would allow the client code to print a log message when it has been disabled, rather than wait for 100 days.

Reply:
100 days was too short.  I changed the default to 100 years and updated the docs in 6251f49.  

For logging the current code will indirectly call the following code when waiting for scan servers.  The Retry code will log something, so we should already be getting logging.

https://github.com/apache/accumulo/blob/eda39bb1c1cf868062743ccc76848c18f62f4916/core/src/main/java/org/apache/accumulo/core/clientImpl/ThriftScanner.java#L269